config.json

```diff
@@ -885,7 +885,12 @@
         "name": "Scale Generator",
         "uuid": "b9c586e8-998b-4f5d-ab98-a08be29a9f19",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "classes",
+          "arrays",
+          "strings",
+          "array-transformations"
+        ],
```

Comment:
My solution : https://exercism.org/tracks/javascript/exercises/scale-generator/solutions/TomPradat 

I found [mstange22's](https://exercism.org/tracks/javascript/exercises/scale-generator/solutions/mstange22) and the [proof solution](https://github.com/exercism/javascript/blob/main/exercises/practice/scale-generator/.meta/proof.ci.js) more concise and understandable so I filled the prerequisites according to these